src/Fmt_ast.ml

```diff
@@ -625,12 +647,14 @@ and fmt_core_type c ?(box= true) ?(in_type_declaration= false) ?pro
   protect (Typ typ)
   @@
   let {ptyp_desc; ptyp_attributes; ptyp_loc} = typ in
+  let c = update_config c ptyp_attributes in
+  maybe_disabled c ptyp_loc
+  @@ fun () ->
```

Comment:
Almost all uses follow this pattern. I wonder if it makes sense to refactor to include`maybe_disabled` into  `update_config` and then write:
```
update_config c attributes
@@ fun c ->
```
instead of:
```
let c = update_config c attributes in
maybe_disabled c loc
@@ fun () ->
```
That might also make it clearer where the config can change but there are no corresponding attributes.

Reply:
that make sense